_data/simple-icons.json

```
@@ -4316,6 +4316,11 @@
             "hex": "2ECE53",
             "source": "https://github.com/docsifyjs/docsify/blob/d01841fd9d829adeeae6cde1d5818ce8798c7e58/docs/_media/icon.svg"
         },
+        {
+            "title": "Doctrine",
+            "hex": "FA6A3C",
```

```suggestion
            "hex": "FC6A31",
```

I'm getting the above from my editor. Where did you find the other one?

---

Honestly, I don't recall but I just checked the color in the SVG and it matches your color suggestion. So, I will make a commit using that color.